src/gameManager.js

```diff
@@ -1,197 +1,248 @@
-import { mobileControls } from './touch.js';
-import { rocket } from './rocket.js';
-import { airfighter } from "./airfighter.js";
-import { deleteEnemies, createEnemies } from './enemy.js';
-import { soundRocketShot, soundMainTheme, soundIntro } from "./music.js";
-import { Step, gameState, fps } from './step.js';
-import { fireGatlingEnemy } from './gatling.js';
-import { resetAmmo } from './ammo.js';
-
-let isGameStarted = false;
-let introduction = document.querySelector('#introduction');
-introduction.addEventListener('click', introductionSkip);
+import { mobileControls } from './touch.js'
+import { rocket } from './rocket.js'
+import { airfighter } from './airfighter.js'
+import { deleteEnemies, createEnemies, enemies } from './enemy.js'
+import {
+  soundRocketShot,
+  soundMainTheme,
+  soundIntro,
+  soundLevelComplete,
+  soundboss,
+} from './music.js'
+import { Step, gameState, fps } from './step.js'
+import { fireGatlingEnemy } from './gatling.js'
+import { resetAmmo } from './ammo.js'
+import { level2boss, createBoss } from './Boss.js'
+import { bossPopup } from './rendering/Helpers.js'
+
+let isGameStarted = false
+let introduction = document.querySelector('#introduction')
+introduction.addEventListener('click', introductionSkip)
 
 const levelState = {
-  isLevelFinished : false,
-  levelNumber : 1,
+  isLevelFinished: false,
+  levelNumber: 0,
```

Comment:
we have levels 1, 2, 3.
We start with level 1.
There is no level 0.

Please change initial level to 1.